.github/actions/windows_dependencies/action.yml

```diff
@@ -29,12 +29,17 @@ runs:
 
     - name: Download Libsodium
       if: steps.check_libsodium.outputs.files_exists == 'false'
+      env:
+        RELEASE_FOLDER: .\libsodium\x64\Release
       shell: cmd
       run: |
         C:\msys64\usr\bin\wget.exe -q https://download.libsodium.org/libsodium/releases/libsodium-${{inputs.libsodium-version}}-msvc.zip
         7z x libsodium-${{inputs.libsodium-version}}-msvc.zip
+        dir %RELEASE_FOLDER% /ad /b /o-n > latest_release_file
```

Comment:
Inside the archive, there are many folders:
```
v100 v110 v120 v140 v141 v142
```
The script should pick one of them. In this PR the mechanism of picking the highest version was introduced.